src/components/TimeSeriesCard/TimeSeriesCard.jsx

```diff
@@ -1,6 +1,6 @@
 import React, { useRef, useMemo, useCallback } from 'react';
 import moment from 'moment/min/moment-with-locales.min';
-import { LineChart } from '@carbon/charts-react';
+import LineChart from '@carbon/charts-react/line-chart';
```

Comment:
Does this just help tree shake the rest of the lib out?

Reply:
unfortunately this is required to trigger the transpilation of the linechart code, won't be needed once 0.16.10 comes out